aws-api/src/main/java/com/amplifyframework/api/aws/SubscriptionOperation.java

```diff
@@ -47,12 +49,14 @@ private SubscriptionOperation(
             @NonNull final OkHttpClient client,
             @NonNull final GraphQLRequest<T> graphQLRequest,
             @NonNull final GraphQLResponse.Factory responseFactory,
-            @NonNull final StreamListener<GraphQLResponse<T>, ApiException> subscriptionListener) {
+            @NonNull final StreamListener<GraphQLResponse<T>, ApiException> subscriptionListener,
```

Comment:
is this in-line with the recent changes that Jameson made to accommodate lambdas?

Reply:
My PR also changes this line, so this will probably generate a merge conflict. One of us (whoever loses the "push first!" race) will have to update.

Reply:
Updated PR to account for this